src/endpoints/createOffersEndpoint.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import OfferModel from '../models/OfferModel'
+import EndpointBuilder from '../EndpointBuilder'
+
+import type { JsonOfferPostType, JsonOfferType } from '../types'
+import Endpoint from '../Endpoint'
+
+export const OFFERS_ENDPOINT_NAME = 'offers'
```

Comment:
I think I need to change this name to offers (as already applied here), as it is used in the webapp. Are there any objections? @maxammann I guess this change requires a new release. How should I do this? :)

Reply:
You are right, next version should be 5.0.1 probably. even though it is breaking.